.github/workflows/build.yml

```diff
@@ -148,9 +148,6 @@ jobs:
       - uses: actions/setup-go@v5
         with:
           go-version: '1.19'
-      - uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: "2.7"
```

Comment:
Anything needed for appveyor configurations?

Reply:
Nope, ruby3.2 is already being setup on lines 151-153. I ran dev canaries and all the tests are passing.